README.md

```diff
@@ -1,5 +1,10 @@
 # M2 Cell
 
+Documentation can be found in the `doc` directory.
+A description of the logic behind some of the original LabView VI's
+can be found in `doc/viDocs`. Much thanks to Patrecio Cortes for
+putting those together.
```

Comment:
"""
A description of the logic behind some of the original LabView VI's can be found in `doc/viDocs`.
Much thanks to Patricio Cortes for putting those together.
"""

Note the line "5". The new sentence should begin at line "6". The name is "Patricio", not "Patrecio".